### PR TITLE
Added animation in texts

### PR DESCRIPTION
**Pull request by coderbysoul**

### CODE REVIEW

**Comment by coderbysoul** [comment]
https://github.com/Codess-Cafe/Codess-Website/assets/96008596/59c53a65-a57b-463d-b5fd-37f282791b27

Issue #28

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
@coderbysoul, this animation does not look good. Can we try something else?

**Comment by coderbysoul** [comment]
Shall I change both the animations?

**Comment by coderbysoul** [comment]
Ok, I will work on it

**Comment by coderbysoul** [comment]
I have modified the text animations for the Come be a part.. line and updated my pull request

https://github.com/Codess-Cafe/Codess-Website/assets/96008596/f96775e1-2d47-4f34-9316-1888f7dac883

File: index.html [hunk 1]
```diff
@@ -213,8 +214,31 @@ <h1 class="big-heading">
                   Community of Empowered Women in Tech
                 </h1>
                 <h5 class="big-heading">
-                  Come be a part of a vibrant community of high achieving,
```
Comment (VidhiBhatt01) [inline index.html:216]: Women not woman

Keep it original only no need to add any animation

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (VidhiBhatt01) [reply index.html:216]: Just keep the first animation on this PR (for the line community of empowered...), no need to add any animation for second line. Keep it original

Comment (coderbysoul) [reply index.html]: Removed the animations from the second line, can you please check

File: app.js [hunk 2]
```diff
@@ -50,4 +50,5 @@ function clearActiveClasses() {
     slides.forEach((slide) => {
         slide.classList.remove('active');
     })
-}
\ No newline at end of file
+}
+
```
Comment (VidhiBhatt01) [inline app.js:54]: nit: extra line

File: index.html [hunk 3]
```diff
@@ -204,7 +204,8 @@
             </li>
           </ul>
         </nav>
-
+        
+
```
Comment (VidhiBhatt01) [inline index.html:208]: nit: extra line

**Comment by VidhiBhatt01** [comment]
Also add a video with only updated changes

File: app.js (hunk 2 above)
Comment (coderbysoul) [reply app.js:54]: I have written the code in app.js from line 55 to 76 for the animations in the second line, which I have deleted now, so this extra line(Line 54) is no longer there. Line 53 is the last line of code in app.js now

File: index.html (hunk 3 above)
Comment (coderbysoul) [reply index.html]: Removed and a made another commit to this pr just now

**Comment by coderbysoul** [comment]
> Also add a video with only updated changes


https://github.com/Codess-Cafe/Codess-Website/assets/96008596/99f8a377-c1f8-4a1d-8adf-9c08653e97fa

Resolved: Issue #34 
Previous: No Animation in the big heading (h1) of hero section
Changes: Added Animation to the big heading(h1) of hero section

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
pull the latest changes and resolve merge conflicts

**Comment by coderbysoul** [comment]
> pull the latest changes and resolve merge conflicts

I have resolved the conflicts in my latest commit

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**